tests/requirements.txt

```diff
@@ -1,4 +1,4 @@
-skale-py==3.0dev8
```

Comment:
Great, but 3.1 is already outdated, sorry for this 😢
Use 3.2 instead (re-check all tests)

Reply:
done